bigtable/tests/system.py

```diff
@@ -57,6 +56,11 @@
 ROUTING_POLICY_TYPE_ANY = 1
 ROUTING_POLICY_TYPE_SINGLE = 2
 EXISTING_INSTANCES = []
+LABEL_KEY = u'python-system'
+label_stamp = datetime.datetime.utcnow() \
+                               .replace(microsecond=0, tzinfo=UTC,) \
+                               .strftime("%Y-%m-%dt%H-%M-%S")
```

Comment:
The [`datetime.datetime.isoformat()`](https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat) method is probably simpler to grasp (and makes the zone offset explicit, too).

Reply:
I couldn't use `datetime.datetime.isoformat()` because it produces format `YYYY-MM-HH'set'HH:MM:SS` and colon `:` is not allowed in the `labels.value`, so I resorted to use `datetime.datetime.strftime()` to have time format be delimited with hyphen `-`.

Reply:
Weird -- I didn't realize there were restrictions on label values (who knows why?).

Reply:
Yeah, label values are weirdly restricted.

> Keys and values can contain only lowercase letters, numeric characters, underscores, and dashes.

https://cloud.google.com/resource-manager/docs/creating-managing-labels

The justification I heard was that the design is to keep labels human-readable and concise for viewing in a list. It's a cloud-wide restriction, so hard to loosen.